build.gradle

```diff
@@ -149,3 +170,10 @@ githubPages {
         }
     }
 }
+
+idea {
```

Comment:
We can remove this code. IntelliJ can resolve additional source sets properly.

Reply:
It might be the case if you import the project as a Gradle project from IntelliJ but it's not the case (and I've just confirmed that by removing this code locally, removing all IntelliJ config files and running `./gradlew openIdea`) if you generate IntelliJ config using Gradle's `idea` plugin. I personally prefer generating IntelliJ configuration from Gradle because it then gives me full control over how IntelliJ sees the project. If you import the project from IntelliJ it seems to sometimes interpret it in a way that is questionable and limiting in my view. So if you are ok with this I'd like to support using the `idea` plugin on this project.

Reply:
Yeah, we can keep it. Nowadays I mostly rely on the native IDEA support for Gradle which works reasonably well.

Reply:
Awesome, thanks for understanding.